egs/librispeech/ASR/zipformer/export-streaming-as-non-streaming-onnx.py

```
@@ -0,0 +1,734 @@
+        """
+        super().__init__()
+        self.encoder = encoder
+        self.encoder_embed = encoder_embed
+        self.encoder_proj = encoder_proj
+
+    def forward2(self, x: torch.Tensor):
+        x_lens = torch.tensor([x.shape[1]], dtype=torch.int32)
+        x, x_lens = self.encoder_embed(x, x_lens)
+        src_key_padding_mask = make_pad_mask(x_lens, x.shape[1]).to(torch.int32)
+        x = x.permute(1, 0, 2)
+        encoder_out, encoder_out_lens = self.encoder(x, x_lens, src_key_padding_mask)
```

Ruff (0.15.15)

[warning] Unpacked variable `encoder_out_lens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)